docs/commands.md

```diff
@@ -29,3 +29,22 @@ when you are happy that it is deleting the right things you run `kops delete clu
 ## `kops version`
 
 `kops version` will print the version of the code you are running.
+
+## Other interesting modes:
```

Comment:
Can we make this section more discoverable?  Maybe in the aws.md guide (even if we then have to duplicate it for gce...)